checks/database_backup_gcp.go

```diff
@@ -32,18 +34,42 @@ func GCPDatabaseBackupCheck(ctx *context.Context, check v1.DatabaseBackupCheck)
 	}
 	var errorMessages []string
 	for _, backup := range backupList.Items {
-		if backup.Status != "SUCCESSFUL" {
-			errorMessages = append(errorMessages, backup.Error.Message)
+		if !(backup.Status == sql.SqlBackupRunStatus_SUCCESSFUL.String() || backup.Status == sql.SqlBackupRunStatus_RUNNING.String() || backup.Status == sql.SqlBackupRunStatus_ENQUEUED.String()) {
```

Comment:
I would rather make it a const list of allowedStatuses that makes it more readable and extendable in the future.

Reply:
const cannot be slices

Reply:
Then I would make it a var slice